src/main/java/de/wejago/hichi2influx/service/JsonSubscriber.java

```diff
@@ -0,0 +1,72 @@
+package de.wejago.hichi2influx.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.influxdb.client.domain.WritePrecision;
+import com.influxdb.client.write.Point;
+import de.wejago.hichi2influx.config.Device;
+import de.wejago.hichi2influx.repository.InfluxDbRepository;
+import java.time.Instant;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections.map.HashedMap;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
```

Comment:
The subscriber should not be a bean managed by Spring, we create it new for every device!